server/app/views/components/UseTag.java

```diff
@@ -0,0 +1,9 @@
+package views.components;
+
+import j2html.tags.ContainerTag;
+
+public final class UseTag extends ContainerTag<UseTag> {
```

Comment:
Add some javadoc with info this tag. It's custom cause I didn't find it in j2html and the SvgTag was custom. Info on `use` can be found here https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Element/use